Wikipedia/Code/SavedArticlesFetcher.m

```diff
@@ -140,9 +140,17 @@ - (void)fetchTitle:(MWKTitle*)title {
         self.fetchOperationsByArticleTitle[title] =
             [self.articleFetcher fetchArticleForPageTitle:title progress:NULL].thenOn(self.accessQueue, ^(MWKArticle* article){
             @strongify(self);
-            return [self downloadImageDataForArticle:article].thenOn(self.accessQueue, ^{
-                [self didFetchArticle:article title:title error:nil];
```

Comment:
huh... so didFetch formerly was never informed of errors? Or did promises deal with that?